.github/ISSUE_TEMPLATE/development-issue.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Development Issue
```

Comment:
What's the "development issue"?

Reply:
How about add a title  "Ask a question" .  Guys may be confused by something, thay can come here to ask for it

Reply:
Ask a question LGTM